openvoxserver/container-entrypoint.d/88-enable-cache-delete-api.sh

```diff
@@ -6,6 +6,6 @@ if [[ "$OPENVOXSERVER_ENABLE_ENV_CACHE_DEL_API" == true ]]; then
   if [[ $(grep 'puppet-admin-api' /etc/puppetlabs/puppetserver/conf.d/auth.conf) ]]; then
     echo "Admin API already set"
   else
-    /opt/puppetlabs/puppet/bin/ruby /docker-entrypoint.d/88-add_cache_del_api_auth_rules.rb
+    /opt/puppetlabs/puppet/bin/ruby /container-entrypoint.d/88-add_cache_del_api_auth_rules.rb
```

Comment:
```suggestion
    my_directory="$(dirname "$0")"
    /opt/puppetlabs/puppet/bin/ruby "${my_directory}/88-add_cache_del_api_auth_rules.rb"
```

Same like https://github.com/OpenVoxProject/container-openvoxserver/pull/59/files#r2242548126